cmd/external-plugins/automated-approver/README.md

```diff
@@ -0,0 +1,61 @@
+#Automated Approver
+
+##Overview
+
+Automated Approver is a tool that allows you to automatically approve pull requests based on the rules you define. It is written in Go. It runs in k8s cluster as a Prow plugin. It reacts on GitHub events. The tool allows automation of the approval process for pull requests in repositories which require reviews before merge. The tool automate PR review process without limiting user write permission on repository. It can provide automated review process for all pull request authors.
```

Comment:
```suggestion
Automated Approver is a tool that allows you to automatically approve pull requests based on the rules you define. The tool allows automation of the approval process for pull requests in repositories which require reviews before merge. The tool automate PR review process without limiting user write permission on repository. It can provide automated review process for all pull request authors.
```

Reply:
I would leave these two sentences out as the next section describes these technical details as well.